microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,225 @@
+
+* `getValues()` for **Bidding's Selection & Key/Value Service:**  This function will run in the Bidding Selection & Key/Value service if the buyer uses this service.  It allows for custom code execution for data lookup and selection.
+
+
+<!-- ------------------------------ -->
+#### Deployment guide
+
+To deploy the Ad Selection API service to Azure:
+
+1. Create an Azure account and create a subscription at [Microsoft Azure](https://azure.microsoft.com).
+
+1. Choose the **Confidential ACO containers** option.
+
+1. Download the Terraform [deployment scripts and guide](https://go.microsoft.com/fwlink/?linkid=2290115).  **Note:** Make sure the hash of the image you downloaded matches the hash in the script you download.
+
+1. The Terraform scripts already include links to public images provided by Microsoft.  Choose the required services and deploy them by following the steps in the included `readme.md`.
+
+   1. Set any specific flags for your environment, including auction worklet locations.
+
+   1. Configure all prerequisites and prepare all tools for deployment.
+
+   1. Create Azure Resources and run terraform script that finish deployment.
+
+1. Develop worklets including `user-defined-function` (UDF) and start Ad Selection API services.
+
+
+<!-- ====================================================================== -->
+## Use the Ad Selection API on your website
+
+To start using the Ad Selection API, sellers and buyers need to use the API, by modifying the JavaScript code of their websites running in Microsoft Edge.
+
+* Sellers: To run private actions, fetch interest groups, and do reporting, update your supply and publisher code.
+
+* Buyers: To create interest groups and store them in the browser, update the code on your advertiser partners' websites.  You can use the delegation mechanism to restrict and allow Interest Group creation on third-party domains.
+
+To learn more about which API methods are available and to review example code, see the [Ad Selection API documentation](https://github.com/WICG/privacy-preserving-ads/blob/main/API%20Details.md).
+
+
+<!-- ====================================================================== -->
+## See also
+<!-- todo: all links in article -->
+
+The end-to-end design and technical documents can be found in the public repo: [Ad Selection API Proposal](https://github.com/WICG/privacy-preserving-ads?tab=readme-ov-file#ad-selection-api-proposal).
+
+Public images that must be used for deployment; only official images from Microsoft will be able to run private auction:
+* Buyer
+   * [buyer-frontend-service](https://mcr.microsoft.com/product/ad-selection/azure/buyer-frontend-service)
```

Comment:
update link text in this section, based on title in destination webpage: per corp style guide, link text should be the destination page's title or heading (or tab-title).